index.html

```diff
@@ -13,21 +13,39 @@
     />
   </head>
   <body>
+    <header class="head">
+      <div class="logo">My logo</div>
+      <nav id="navbar1"  onclick="showNav">
+        <img src="image/Union.png" alt="button" />
+      </nav>
+      <nav class="navbar">
+        <ul class="container">
+          <li><a href="#facebook">Portfolio</a></li>
+          <li><a href="#About">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <!-- ----------------- hamburger ------------------- -->
+    <div class="ham" id="burger">
```

Comment:
- [ ] Kindly note that when I open the the project in the mobile version, I the mobile menu is visible already, I think it will be good nice if the mobile menu is hidden when the project opens in mobile menu, add be visible once I click on the humberger button.


https://github.com/VivekRaaZ/portfolio/assets/65068771/98b6989b-db07-4933-b65a-e1cbb899a950